javaslang/src/main/java/javaslang/collection/Array.java

```diff
@@ -1059,6 +1059,10 @@ public int length() {
         return Collections.scanRight(this, zero, operation, Array::ofAll);
     }
 
+    @Override public Array<T> shuffle() {
```

Comment:
Not sure how I keep missing these. I've installed the `idea-settings.jar` but something seems to be wrong with the IDE...

Will fix!